app/page.tsx

```diff
@@ -3,11 +3,36 @@
 import React, { useState } from "react";
 import Link from "next/link";
 import states from "@/data/states";
-import { Input, Card, CardBody } from "@nextui-org/react";
+import {
+  Input,
+  Card,
+  CardBody,
+  Button,
+  Select,
+  SelectItem,
+} from "@nextui-org/react";
+import { useRouter } from "next/navigation";
+import { Form } from "@nextui-org/form";
+
+const currentYear = new Date().getFullYear();
+
+export const years = Array.from(
```

Comment:
You cant export from a Nextjs page. Running a build will return this error: 
```
app/page.tsx
Type error: Page "app/page.tsx" does not match the required types of a Next.js Page.
  "years" is not a valid Page export field.
Error: Command "npm run build" exited with 1
```

If you are using `years` outside of the scope of this page, then consider moving it to `app/data` and export it from there! @JonathanTylerWeber